### PR TITLE
#436 Fixed positioning of import labels

### LINKED ISSUE #436: Import buttons positioning in import menu

### PR DESCRIPTION
**Pull request by Dacaspex**
Fixes #436 
#### What have I done
- Added a larger breakpoint for the text next to the import navigation buttons.
- Added a tooltip to each navigation button.

#### Testing
Open the import screen, resize the window and see that it looks alright.

#### Addresses SRD section(s)
None

### CODE REVIEW

File: src/app/import/import.component.html [hunk 1]
```diff
@@ -81,16 +81,18 @@ <h5 class="mb-1">{{ modelPreview.name }}</h5>
                 <a
                     (click)="changeState(StartStateEnum.ImportManual)"
                     class="btn btn-link cursor-pointer"
+                    placement="left"
+                    ngbTooltip="Select files manually"
```
Comment (Chessnl) [inline src/app/import/import.component.html]:
```suggestion
                    ngbTooltip="Import files manually"
```

File: src/app/import/import.component.html [hunk 2]
```diff
@@ -81,16 +81,18 @@ <h5 class="mb-1">{{ modelPreview.name }}</h5>
...
                 >
-                    <span class="d-none d-sm-block">Select files manually</span>
                     <fa-icon class="ml-1" [icon]="plusIcon"></fa-icon>
```
Comment (Chessnl) [inline src/app/import/import.component.html]: I personally like [fa-file-import](https://fontawesome.com/icons/file-import?style=solid) more for this functionality,

File: src/app/import/import.component.html [hunk 3]
```diff
@@ -81,16 +81,18 @@ <h5 class="mb-1">{{ modelPreview.name }}</h5>
                 <a
```
Comment (Chessnl) [inline src/app/import/import.component.html]: I would suggest adding a tooltip to the arrowLeftIcon as well. As it is unclear why only 2 of the 3 would be undefined, while it is not clear what the third's purpose is. I would probably add a tooltip of 'To welcome'. Similarly, should a tooltip be added to this button on the import screen as well.

Comment (Dacaspex) [reply src/app/import/import.component.html]: Added

File: src/app/import/import.component.html (hunk 2 above)
Comment (Dacaspex) [reply src/app/import/import.component.html]: Changed

File: src/app/import/import.component.html (hunk 1 above)
Comment (Dacaspex) [reply src/app/import/import.component.html]: Changed

File: src/app/import/import.component.html [hunk 4]
```diff
@@ -166,7 +170,12 @@ <h5 class="mb-1">{{ modelPreview.name }}</h5>
         <!-- Top navigation buttons, only visible when on the import manual page -->
         <div *ngIf="isImportManual()">
             <div class="start-top-button-row d-flex justify-content-between text-white p-3">
-                <a (click)="changeState(StartStateEnum.Import)" class="btn btn-link cursor-pointer">
+                <a
+                    (click)="changeState(StartStateEnum.Import)"
+                    class="btn btn-link cursor-pointer"
+                    placement="right"
+                    ngbTooltip="Back"
```
Comment (Chessnl) [inline src/app/import/import.component.html]:
```suggestion
                    ngbTooltip="To model import"
```

Comment (Chessnl) [reply src/app/import/import.component.html]: Suggestion. Does not have to be accepted. After being accepted or rejected, can be merged.

**Pull request merged**